rest-api/model/calendar.py

```diff
@@ -0,0 +1,8 @@
+from model.base import Base
+from sqlalchemy import Date, Column
+
```

Comment:
Nit: remove extra line here

Reply:
Done.  (My IDE complains about deviating from PEP8, but I see this is local convention.)